mastering-plone/add-ons.md

```diff
@@ -2,8 +2,7 @@
 
 # Extending Plone With Add-on Packages
 
-````{sidebar}
-Classic chapter
+````{sidebar} Classic chapter
```

Comment:
I think I had parsing problems with `{sidebar}`, maybe not, can't remember for sure. If I recall correctly, I found that moving the content of some directives from the same line to the next line fixed the parsing problems.

Reply:
"Classic chapter" / "Volto chapter" is meant to be sidebar title (rendered bold and increased font size). It is regarded as side bar title only if it is on the same line as directive sidebar. So I prefer ````{sidebar} Classic chapter. It renders well without error or warning.
![sidebar_after_merged_and_edited](https://user-images.githubusercontent.com/1144737/132134888-e3b8d089-087e-4fab-8a61-21695992b5a1.png)